### PR TITLE
Feat/implement notification

### PR DESCRIPTION
**Pull request by tobslob**
### Implement Dashboard Notification

### CODE REVIEW

File: src/pages/Notification/Notification.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+
+.notification-page {
+    width: rem(420px);
+    height: rem(400px);
+    background-color: white;
+    position: absolute;
+    right: rem(30px);
+    margin: rem(70px) rem(120px);
+    border-radius: rem(20px);
```
Comment (tejiri4) [inline src/pages/Notification/Notification.scss:13]: add  z-index: 1; here. nice work

**Comment by tejiri4** [comment]
you can merge after adding the z-index

File: .env [hunk 2]
```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```
Comment (tejiri4) [inline .env]: you can add this to gitignore

File: src/pages/Notification/Notification.scss (hunk 1 above)
Comment (tobslob) [reply src/pages/Notification/Notification.scss:13]: Done. Thanks

**Pull request merged**